_getting_started/installation/index.md

```diff
@@ -4,35 +4,120 @@ layout: default
+* [Windows](#windows)
+* [Linux](#linux)
 
-There appears to be an issue with Cabal that is affecting installation.  If you are unable to install TidalCycles with the directions below, please try these alternate installation instructions.
+<a id="macos">&nbsp;</a>
+<hr class="marker">
 
-From a terminal window, run this:
+## Mac OS 
+
+### 1. Prerequisites
+
+You first need to install the following three pieces of software,
+Haskell, Atom, and SuperCollider. You can download them from their
+websites - click on each of the below:
+
+* [Haskell](https://www.haskell.org/platform/)
+* [Atom](https://atom.io/)
+* [SuperCollider](http://supercollider.github.io/download) (version 3.7 or later)
+* You may also need to install [Git](https://git-scm.com/)
+
+You will need the SuperCollider sc3-plugins for using many of the synths included in 
+SuperDirt. Most of the examples in the documentation will still work, so you could
+skip this step and return to it later. You can install the latest version from
+[github](https://github.com/supercollider/sc3-plugins) according to
+the instructions there.
+
+### 2. Install the TidalCycles pattern engine
+
+>**NOTE** As of September 26 2017, There appears to be an issue with Cabal that is 
+>affecting installation on MacOS.  
+>If you are unable to install TidalCycles with the directions below, please try 
+>these alternate installation instructions. From a terminal window, run this:
+
+>~~~~bash
+>stack setup
+>stack install tidal
+>~~~~
+
+>Once that's done, you'll also need to change a configuration setting
+>in Atom. Find the package settings for TidalCycles (*preferences >
+>tidalcycles > settings*). Change the ```'ghci path'``` option to this:
+
+>```stack ghc -- --interactive```
+
+>Restart Atom and it should be able to start TidalCycles.
+
+>**Skip ahead to MacOS Step #3 (Install the SuperDirt synth) if you used this method**
+
+#### Standard MacOS Instructions:
+
+Open a Terminal window and type in the following command, to install
+TidalCycles:
 
 ~~~~bash
-stack setup
-stack install tidal
+cabal install tidal
+~~~~
+
+> If you're unsure how to open a terminal window, in Mac OS it's 
+> "Terminal" under Utilities.
+
+
+### 3. Install the SuperDirt synth
+
+Start SuperCollider, and in the editor window paste in the following line of code:
+
+~~~~c
+include("SuperDirt")
+~~~~
+
+Run the code by clicking on it, to make sure the cursor is on this
+line, then hold down shift and press enter.
+
+It will download SuperDirt and
+you will see it has completed when the Post Window displays:
+
+~~~~c
+... the class library may have to be recompiled.
+-> SuperDirt
 ~~~~
 
-Once that's done, you'll also need to change a configuration setting
-in Atom. Find the package settings for TidalCycles (```preferences >
-tidalcycles > settings```). Change the ```'ghci path'``` option to this:
+After it has completed, you will need to restart SuperCollider (or
+alternatively, recompile the class library via the "Language" menu).
```

Comment:
This is a good point to mention that people can optionally copy the SuperDirt `startup.scd` template into their main `startup.scd` if they want to auto-launch by default.

Reply:
Where is the SuperDirt `startup.scd` located? I like this idea but I don't know of this file.

Reply:
ah, found it: https://github.com/musikinformatik/SuperDirt/blob/master/superdirt_startup.scd

Reply:
I wonder if it would be better to put the mention of `superdirt_startup.scd` into the "Running Tidalcycles" section on the site that immediately follows the installation instructions. e.g. where it says to run `SuperDirt.start`.

Reply:
Yeah maybe this is better in a different place, and we could link to it here?